schemas/authenticate-hawk-response.yml

```diff
@@ -0,0 +1,45 @@
+$schema:  http://json-schema.org/draft-04/schema#
+title:                    "Hawk Signature Authentication Response"
+description: |
+  Response from a request to authenticate a hawk request.
+anyOf:
```

Comment:
I'm not sure this schema is elegant... Perhaps I need to add a `status: "authorized" || "unauthorized"` to make it easy to distinguish between the two objects for `anyOf`. Thoughts?

Mostly I focus on keeping it future proof where we can support a different authentication scheme.